src/world_interface/simulator_interface.cpp

```diff
@@ -364,9 +368,9 @@ DataPoint<pose_t> readVisualOdomVel() {
 	return DataPoint<pose_t>{};
 }
 
-DataPoint<double> readIMUHeading() {
-	std::lock_guard<std::mutex> guard(headingMutex);
-	return lastHeading;
+types::DataPoint<navtypes::eulerangles_t> readIMU() {
```

Comment:
Same as above. Only threw me off because the same file has `DataPoint` for `lastAttitude` that does not compile if you change it to `types::DataPoint` because it's not in the `robot` namespace.

FYI these are nitpicky and you can ignore if you want. They just throw off the consistency and requires reading more context around the code to see where you need to qualify and not qualify.